Core/Date.cs

```diff
@@ -30,315 +30,138 @@ namespace System
 	[Serializable]
 	public struct Date : IComparable, IFormattable, ISerializable, IComparable<Date>, IEquatable<Date>
 	{
-		private DateTime _dt;
+		private readonly DateTime _dt;
 
 		public static readonly Date MaxValue = new Date(DateTime.MaxValue);
 		public static readonly Date MinValue = new Date(DateTime.MinValue);
 
-		public Date(int year, int month, int day)
-		{
-			_dt = new DateTime(year, month, day);
-		}
+		public Date(int year, int month, int day) => _dt = new DateTime(year, month, day);
 
-		public Date(DateTime dateTime)
-		{
-			_dt = dateTime.AddTicks(-dateTime.Ticks % TimeSpan.TicksPerDay);
-		}
+		public Date(DateTime dateTime) => _dt = dateTime.AddTicks(-dateTime.Ticks % TimeSpan.TicksPerDay);
 
-		private Date(SerializationInfo info, StreamingContext context)
-		{
-			_dt = DateTime.FromFileTime(info.GetInt64("ticks"));
-		}
+		private Date(SerializationInfo info, StreamingContext context) => _dt = DateTime.FromFileTime(info.GetInt64("ticks"));
 
-		public static TimeSpan operator -(Date d1, Date d2)
-		{
-			return d1._dt - d2._dt;
-		}
+		public static TimeSpan operator -(Date d1, Date d2) => d1._dt - d2._dt;
 
-		public static Date operator -(Date d, TimeSpan t)
-		{
-			return new Date(d._dt - t);
-		}
+		public static Date operator -(Date d, TimeSpan t) => new Date(d._dt - t);
 
-		public static bool operator !=(Date d1, Date d2)
-		{
-			return d1._dt != d2._dt;
-		}
+		public static bool operator !=(Date d1, Date d2) => d1._dt != d2._dt;
 
-		public static Date operator +(Date d, TimeSpan t)
-		{
-			return new Date(d._dt + t);
-		}
+		public static Date operator +(Date d, TimeSpan t) => new Date(d._dt + t);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement alternative method 'Add' for the operator '+'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)